action/cache.go

```diff
@@ -21,3 +23,17 @@ func CacheClear() {
 
 	msg.Info("Glide cache has been cleared.")
 }
+
+// CachePath prints the path to a cached dependency's sources.
+func CachePath(importPath string) {
+	for _, d := range EnsureConfig().Imports {
```

Comment:
`EnsureConfig` doesn't load the lock file. That means it won't find transitive dependencies. It might be a good idea to load a lock, if present, and fall back to config if the lock is not there.